v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -22,7 +22,7 @@ const {
 
 const hashEncrypt = object => {
   Object.keys(object).forEach(key => {
-    if (hashAttributes.includes(key)) {
+    if (hashAttributes.includes(key) && object[key]) {
```

Comment:
@ItsSudip Please add test cases for this.

Reply:
Are you asking for unit test cases?

Reply:
Yes